feature/experimental/gribi/otg_tests/backup_nhg_action/backup_nhg_action_test.go

```diff
@@ -415,7 +415,9 @@ func testDecapEncap(ctx context.Context, t *testing.T, args *testArgs) {
 	}
 	t.Run("ValidateDecapPath", func(t *testing.T) {
 		t.Log("Validate traffic after decap is recieved on port4 and no traffic on other flows/ate ports")
-		validateTrafficFlows(t, args.ate, []gosnappi.Flow{decapFLow}, []gosnappi.Flow{baseFlow, encapFLow}, decapFlowFliter)
+		if !deviations.SecondaryBackupPathTrafficFailover(ondatra.DUT(t, "dut")) {
```

Comment:
Reuse args.dut here instead of making a call to ondatra.

Reply:
Updated with args.dut as suggested.